examples/cellar/gen/http/storage/server/types.go

```diff
@@ -229,7 +229,9 @@ func NewAddBottle(body *AddRequestBody) *storage.Bottle {
 		Description: body.Description,
 		Rating:      body.Rating,
 	}
-	v.Winery = unmarshalWineryRequestBodyToWinery(body.Winery)
+	if body.Winery != nil {
```

Comment:
Are these new "if" statement expected? was the code wrong before?

Reply:
`body.Winery` is a pointer field. I think it would be safer to check for nil even though Winery is a required attribute.